RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -46,6 +52,7 @@ public struct CustomerCenterView: View {
             } else {
                 if let configuration = self.viewModel.configuration {
                     destinationView(configuration: configuration)
+                        .environment(\.localization, configuration.localization)
```

Comment:
Ah I added the view model to the environment below... But I think this is ok and I've been thinking about removing the view model, so I think it's ok even if they are in separate places now.

Reply:
yes, I thought about that as well! but concluded it belongs here because here it is where we have the configuration object available